### ISSUE TITLE
vue-loader:  11.3.4 → 12.0.0 breaks <style lang="scss">

### ISSUE THREAD

**Issue by bentolor** [vuejs/vue-loader#790]
### Version
12.0.0

### Reproduction link
Can't reproduce a build problem

### Steps to reproduce
After upgrading my `vue-loader` dependency from 11.3.4 to 12.0.0 the `<style lang="scss"> does no longer work. 

I'm using the latest vue-cli webpack template with the following added to `webpack.base.conf`

````
      {
        test: /\.scss$/,
        loaders: [ 'style', 'css', 'sass' ]
      },
````

### What is expected?
It should work as previously.

### What is actually happening?
````
ERROR  Failed to compile with 1 errors                                                                                                                 19:16:22

 error  in ./src/App.vue

Module build failed: 
.wrapper {
         ^
      Invalid CSS after ".wrapper {": expected "}", was "{"
      in /home/ben/projects/mfusst-ui.git/app/src/App.vue (line 18, column 11)

 @ ./~/vue-style-loader!./~/css-loader?{"minimize":false,"sourceMap":false}!./~/vue-loader/lib/style-compiler?{"vue":true,"id":"data-v-bd26cd6c","scoped":false,"hasInlineConfig":false}!./~/sass-loader/lib/loader.js?{"indentedSyntax":true,"sourceMap":false}!./~/vue-loader/lib/selector.js?type=styles&index=0!./src/App.vue 4:14-404 13:3-17:5 14:22-412
 @ ./src/App.vue
 @ ./src/main.js
 @ multi ./build/dev-client ./src/main.js
````

**Comment by TheLarkInn** [comment]
Am also seeing this as well. @yyx990803 let me know if you need some reproducibles.

**Comment by yyx990803** [comment]
Fixed in 12.0.1!
